packages/eds-core-react/src/components/Button/Button.stories.tsx

```diff
@@ -422,3 +437,86 @@ GroupSplit.decorators = [
     </Stack>
   ),
 ]
+
+export const ToggleExclusiveSelection: Story<ToggleButtonProps> = () => {
```

Comment:
Still seems a bit weird that the story starts off with nothing selected, when the rule is "one should always be selected"